types/react-dropzone/react-dropzone-tests.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Dropzone = require('react-dropzone');
+import * as Dropzone from 'react-dropzone';
```

Comment:
While the changes to _index.d.ts_ are ok, I would caution you against using the ES6-style `import * as X from ...` syntax, as it is not designed to work with `export =` in this fashion.